src/collectd.conf.pod

```diff
@@ -7140,6 +7141,10 @@ than B<Interval> defined globally.
 The B<Query> block identifies a query to execute against the redis server.
 There may be an arbitrary number of queries to execute.
 
+=item B<Db> I<Db>
```

Comment:
Database is more clear than Db, don't you think?

Reply:
Agreed, we use full word in other places.